homeassistant/components/viam/strings.json

```diff
@@ -0,0 +1,168 @@
+        "description": "Provide the credentials for communicating with the Viam service.",
+        "data": {
+          "api_id": "API Key ID",
+          "api_key": "API Key Value",
+          "address": "Robot Address",
+          "secret": "Robot Secret"
+        },
+        "data_description": {
+          "address": "Find this under the Code Sample tab in the app.",
+          "secret": "Find this under the Code Sample tab in the app when 'include secret' is enabled."
+        }
+      },
+      "robot": {
+        "data": {
+          "robot": "Select a robot"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "selector": {
+    "credential_type": {
+      "options": {
+        "api-key": "Org API Key",
+        "robot-location-secret": "Robot Location Secret"
+      }
+    }
+  },
+  "exceptions": {
+    "camera_entity_not_found": {
+      "message": "A camera entity called {camera_name} was not found"
+    },
+    "camera_not_found": {
+      "message": "A camera for the entity {camera_name} was not found"
+    },
+    "robot_credentials_required": {
+      "message": "The robot location and secret are required for this connection type."
+    },
+    "robot_credentials_not_found": {
+      "message": "The necessary credentials for the RobotClient could not be found."
+    }
+  },
+  "services": {
+    "capture_data": {
+      "name": "Capture data",
+      "description": "Send arbitrary tabular data to Viam for analytics and model training.",
+      "fields": {
+        "values": {
+          "name": "Values",
+          "description": "List of tabular data to send to Viam."
+        },
+        "component_name": {
+          "name": "Component Name",
+          "description": "The name of the configured robot component to associate with the data."
+        },
+        "component_type": {
+          "name": "Component Type",
+          "description": "The type of the associated component."
+        }
+      }
+    },
+    "capture_image": {
+      "name": "Capture image",
+      "description": "Send images to Viam for analytics and model training.",
+      "fields": {
+        "filepath": {
+          "name": "Filepath",
+          "description": "Local file path to the image you wish to upload."
+        },
+        "camera": {
+          "name": "Camera Entity",
+          "description": "The camera entity from which an image is captured."
+        },
+        "file_name": {
+          "name": "File Name",
+          "description": "The name of the file that will be displayed in the metadata within Viam."
+        },
+        "component_name": {
+          "name": "Component Name",
```

Comment:
Please reference the above text to avoid duplicates. Please adopt the other lines too